Remotion/Core/Core/Utilities/StringUtility.cs

```diff
@@ -275,9 +275,12 @@ public static string[] ListToStringArray (IList list)
 
     public static string ConcatWithSeparator (IList list, string separator)
     {
+#pragma warning disable 618
       return ConcatWithSeparator (list, separator, null, null);
+#pragma warning restore 618
     }
 
+    [Obsolete ("Use ConcatWithSeperator (IList, string) instead. Parameter 'format' is no longer used.")]
     public static string ConcatWithSeparator (IList list, string separator, string format, IFormatProvider formatProvider)

```

Comment:
Imo another overload should be added, which has the `format` parameter removed.
`public static string ConcatWithSeparator (IList list, string separator, IFormatProvider formatProvider)`

The 2 calls of the old method enclosed in `pragma` as well as the obsolete method should then call the method with the new signature. Or is this another issue? @MichaelKetting, quoting https://github.com/re-motion/Trunk-SVN2Git/pull/75/files#r428597852

Reply:
since there's no one using this, this helper should just be removed. hence the obsolete message for now. Since there's a ton of no longer needed legacy stuff on StringUtility, a clean-up should be a coordinated task, not just a single method.

Reply:
Got it

Reply:
Added https://re-motion.atlassian.net/browse/RM-7436